central/resourcecollection/datastore/datastore_impl.go

```diff
@@ -54,16 +54,29 @@ func (ds *datastoreImpl) initGraph() error {
 			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
 			sac.ResourceScopeKeys(resources.WorkflowAdministration)))
```

Comment:
This can just be `sac.WithAllAccess(context.Background())` since we know we are triggering the access and for what purposes.

Reply:
addressed